src/persist-client/src/read.rs

```diff
@@ -102,14 +104,14 @@ where
         trace!("SnapshotIter::next timeout={:?}", timeout);
         let deadline = Instant::now() + timeout;
         loop {
-            let key = match self.batches.last() {
+            let batch = match self.batches.last() {
```

Comment:
nit: destructure this into key and desc instead of using the .0 and .1 below please